src/worker/api/index.ts

```diff
@@ -494,6 +496,11 @@ const createLeague = async ({
 		"challengeFiredMissPlayoffs",
 		challengeFiredMissPlayoffs,
 	);
+	upsertGameAttribute(
```

Comment:
You're gonna hate me, but this won't merge cleanly as soon as I merge what I'm working on now :)

On the plus side, it's a very easy change. Code is actually simpler now. The existence of that `upsertGameAttribute` function is absurd.

Reply:
Now it just needs to go in here: https://github.com/dumbmatter/gm-games/blob/master/src/worker/api/index.ts#L428-L439 no more `upsertGameAttribute`